### PR TITLE
feat(dev-tools): Add dark mode & restructure files

### PR DESCRIPTION
**Pull request by td**
Had a quick play on my lunch break with the extension:
 - Updated the folder structure of the extension
 - Added a dark mode toggle in the popup window

### CODE REVIEW

File: css/chrome_style.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+.toggle input[type='checkbox']:checked ~ label {
+    background-color: #00a0fc;
+    border-color: #006dc9;
+}
+
+.toggle input[type='checkbox']:checked ~ label::after {
+    background-color: #0054b0;
+    transform: translateX(25px);
+}
```
Comment (casserlyprogramming) [inline css/chrome_style.css]: empty line

**Review by casserlyprogramming — APPROVED** [review]

**Comment by casserlyprogramming** [comment]
Nice!

**Comment by casserlyprogramming** [comment]
Worth mentioning that if you already have a lists.js you will need to move it into the js folder and to get the dark mode to work you need to go to `chrome://extensions` and reload the plugin.

**Comment by td** [comment]
> Worth mentioning that if you already have a lists.js you will need to move it into the js folder and to get the dark mode to work you need to go to `chrome://extensions` and reload the plugin.

I updated the readme to add the js/ folder to the instructions but an extension reload is required for the permissions to update as well.

**Pull request merged**